pkg/es/writer/esBulkHandler.go

```diff
@@ -318,3 +319,56 @@ func PostBulkErrorResponse(ctx *fasthttp.RequestCtx) {
 	responsebody["status"] = 400
 	utils.WriteJsonResponse(ctx, responsebody)
 }
+
+func ProcessDeleteIndex(ctx *fasthttp.RequestCtx, myid uint64) {
+	inIndexName := utils.ExtractParamAsString(ctx.UserValue("indexName"))
+
+	convertedIndexNames, indicesNotFound := deleteIndex(inIndexName, myid)
+
+	if indicesNotFound == len(convertedIndexNames) {
+		ctx.SetStatusCode(fasthttp.StatusNotFound)
+		response := make(map[string]interface{})
+		final := make(map[string]interface{})
+		var items = make([]interface{}, 0)
+		items = append(items, *utils.NewDeleteIndexErrorResponseInfo(inIndexName))
+		final["root_cause"] = items
+		final["error"] = *utils.NewDeleteIndexErrorResponseInfo(inIndexName)
+		final["status"] = 404
+		response["error"] = final
+		utils.WriteJsonResponse(ctx, response)
```

Comment:
Why is there this extra structure of `final.error`, `final.status`, `final.root_cause`? Why don't we have only `response.error` like we do in most error handling for other APIs? (It could be fine to keep this structure, I'm just not sure why we have it)

Reply:
removed extra structure